upsun/10-sulu-upsun.yaml

```diff
@@ -0,0 +1,209 @@
+requirements:
+  - type: file_exists
+    value: composer.json
+  - type: has_composer_package
+    value: sulu/sulu
+
+template: |
+    {{- $hasVarnish := false -}}
+    {{- range $service := $.Services -}}
+        {{- if eq $service.Type "varnish" -}}
+            {{- $hasVarnish = true -}}
+        {{- end -}}
+    {{- end -}}
```

Comment:
You should be able to merge the two loops over `$.Services` and move `routes` after `services` (because keys order in the YAML does not matter)

Reply:
Updated to use a single initial loop to gather the needed information. I’d prefer to keep the original order in the YAML file for better DX and readability.